### PR TITLE
[TOR-4021] Navbar popover styling

### PR DESCRIPTION
**Pull request by ashraj98**
[TOR-4021](https://torchdental.atlassian.net/browse/TOR-4021)

Description:
Changes to the navbar for new cart nav item location and new popover style for navbar main menu.

Without popover style (maintaining current design):
Screen Shot 2021-09-13 at 2 24 36 PM
Screen Shot 2021-09-13 at 2 18 44 PM
Screen Shot 2021-09-13 at 2 18 53 PM
Screen Shot 2021-09-13 at 2 24 29 PM

With popover (new styling):
Screen Shot 2021-09-13 at 2 23 56 PM
Screen Shot 2021-09-13 at 2 24 04 PM
Screen Shot 2021-09-13 at 2 24 54 PM
Screen Shot 2021-09-13 at 2 25 02 PM

### CODE REVIEW

File: src/sass/_icons.scss [hunk 1]
```diff
@@ -1136,6 +1136,15 @@
       @import 'icons/cart.svg';
     }
   }
+  &--badge,
+  &--badge:hover,
+  &--badge:active {
+    &::before {
+      height: $s16;
+      $icon-color: #fff;
```
Comment (pyen) [inline src/sass/_icons.scss:1144]: `$color--white` ?

File: src/sass/components/_badge.scss [hunk 2]
```diff
@@ -40,3 +40,12 @@
...
 }
+
+.badgeCart {
+  @extend .badge;
+  background-color: $color--secondary-active;
+  min-width: $s40;
+  height: $s32;
+  font-size: $s16;
+  cursor: inherit;
+}
```
Comment (pyen) [inline src/sass/components/_badge.scss:51]: IMO, the cart icon badge could use some x-padding. The "1" is close to the edge in the screenshots.

File: src/sass/components/_nav.scss [hunk 3]
```diff
@@ -207,6 +216,49 @@
...
+              width: 0;
+              height: 0;
+            }
+          }
+        }
+        @include desktop-breakpoint {
+          .menu {
+            &-body {
+              position: fixed;
+              width: 308px;
```
Comment (pyen) [inline src/sass/components/_nav.scss:241]: Is this a derived value? Perhaps express as an equation instead?

**Review by pyen — APPROVED** [review]

File: src/sass/components/_nav.scss (hunk 3 above)
Comment (ashraj98) [reply src/sass/components/_nav.scss:241]: This is from the design direction from the figma mocks. I saw the standard `.menu-body {` was also hardcoded to `270px` so I hardcoded this as well.

**Pull request merged**